core/renderers/geras/drawer.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2019 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Renderer that preserves the look and feel of Blockly pre-2019.
+ * @author fenichel@google.com (Rachel Fenichel)
+ */
+'use strict';
+
+goog.provide('Blockly.geras.Drawer');
+
+goog.require('Blockly.blockRendering.ConstantProvider');
+goog.require('Blockly.blockRendering.Drawer');
+goog.require('Blockly.geras.Highlighter');
+goog.require('Blockly.geras.RenderInfo');
+
+
+/**
+ * An object that draws a block based on the given rendering information.
+ * @param {!Blockly.BlockSvg} block The block to render.
+ * @param {!Blockly.geras.RenderInfo} info An object containing all
+ *   information needed to render this block.
+ * @package
+ * @constructor
+ * @extends {Blockly.blockRendering.Drawer}
+ */
+Blockly.geras.Drawer = function(block, info) {
+  Blockly.geras.Drawer.superClass_.constructor.call(this, block, info);
+  // Unlike Thrasos, Geras has highlights and drop shadows.
+  this.highlighter_ = new Blockly.geras.Highlighter(
+      /** @type {!Blockly.geras.RenderInfo} */ (this.info_), this.pathObject_);
```

Comment:
Why the cast if the type of the info parameter is the same?

Reply:
Actually, I can avoid the cast if I pass in `info` instead of `this.info_`.
`this.info_` is set in the superclass constructor so it's only annotated as a `Blockly.blockRendering.RenderInfo`.

Reply:
Yep, sounds good.